core/templates/dev/head/pages/collection_editor/collection_editor.html

```diff
@@ -117,6 +117,7 @@
 
   {# TODO(bhenning): Remove these once RouterServices is decoupled from the exploration editor #}
   <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/EditorServices.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/pages/exploration_editor/ExplorationWarningsService.js"></script>
```

Comment:
@nithusha21, Remove this as it's not required to import  `ExplorationWarningsService' here.

Reply:
ok will do